### DISCUSSION TITLE
Getting error on bash

### DISCUSSION THREAD

**Discussion by 0xthem7** [whoami-anoint/automation#8] [Ideas]
Probe is an awesome tool but getting some errors on bash. It would be charm if it run on docker...

**Comment by whoami-anoint** [comment]
Good point. We'll do looking forward to this **feature.**

**Reply by 0xthem7** [reply]
What about making it online, like running on a server and provide a web version ?

**Reply by whoami-anoint** [reply]
Dear @m7-Selenophile , nice question. For your concern, we like to announce that we can run our tool in VPS for smooth performance in recon and we're planning to make features that works as a web version.

**Reply by 0xthem7** [reply]
@whoami-anoint  Looking forward for web version
